app/actors/hyrax/actors/collections_membership_actor.rb

```diff
@@ -12,28 +12,29 @@ class CollectionsMembershipActor < AbstractActor
       # @param [Hyrax::Actors::Environment] env
       # @return [Boolean] true if create was successful
       def create(env)
-        attributes_collection = env.attributes.delete(:member_of_collections_attributes)
-        assign_nested_attributes_for_collection(env, attributes_collection) &&
-          next_actor.create(env)
+        assign_nested_attributes_for_collection(env) && next_actor.create(env)
       end
 
       # @param [Hyrax::Actors::Environment] env
       # @return [Boolean] true if update was successful
       def update(env)
-        attributes_collection = env.attributes.delete(:member_of_collections_attributes)
-        assign_nested_attributes_for_collection(env, attributes_collection) &&
-          next_actor.update(env)
+        assign_nested_attributes_for_collection(env) && next_actor.update(env)
       end
 
       private
 
+        ##
         # Attaches any unattached members.  Deletes those that are marked _delete
         # @param [Hash<Hash>] a collection of members
-        def assign_nested_attributes_for_collection(env, attributes_collection)
-          return true unless attributes_collection
+        # rubocop:disable Metrics/MethodLength
```

Comment:
The rubocop disables explicitly support the call to `emit_deprecation`. When the dual behavior is removed this will handily pass all Rubocop's ABC and MethodLength checks.